components/badge/index.tsx

```diff
@@ -100,15 +103,15 @@ export default class Badge extends React.Component<BadgeProps, any> {
     }
 
     const scrollNumber = hidden ? null : (
-      <ScrollNumber
+      (wrapBadge || (_ => _))(<ScrollNumber
```

Comment:
can we use defaultProps for wrapBadge instead?

Reply:
Thought about this too, but having a function in `defaultProps` that does nothing felt strange for me and I also thought it might add issues with TypeScript (I had some problems with TS 3.0.0 which adds some kine of `defaultProps` handling)?!